src/org/traccar/protocol/TelicProtocolDecoder.java

```diff
@@ -105,4 +110,57 @@ protected Object decode(
         return position;
     }
 
+    private String decodeAlarm(String event) {
+
+        int eventId = 0;
+
+        try {
+            eventId = Integer.parseInt(event);
+        } catch (Exception e) {
```

Comment:
It can't fail because we use regular expression, so I don't think we need to catch anything. We can just use `parser.nextInt()` and pass int value directly into `decodeAlarm` method.